contrib/babelfishpg_tsql/src/properties.c

```diff
@@ -168,8 +170,14 @@ Datum serverproperty(PG_FUNCTION_ARGS) {
 	}
     else if (strcasecmp(property, "Collation") == 0)
     {
+		#if 0
 		char *ret = pstrdup(pltsql_server_collation_name);
 		vch = tsql_varchar_input(ret, strlen(ret), -1);
+		#endif
+		if (!server_collation_name)
+			server_collation_name = GetConfigOption("babelfishpg_tsql.server_collation_name", true, false);
```

Comment:
We should not cache the value from `GetConfigOption` (Ref: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/543).

Reply:
Updated in latest revision